src/canvas.class.js

```diff
@@ -860,38 +858,37 @@
     /**
      * @private
      */
-    _isLastRenderedObject: function(e) {
+    _isLastRenderedObject: function(pointer) {
```

Comment:
this function is a shortcut to check last selected target before all. But why just with controls above overaly? i think it should be checked always.